### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adammw**
:v:

/cc @zendesk/vegemite @alieu

### Description

Add info on how to build the app for uploading.

### CODE REVIEW

**Comment by sandlerr** [comment]
foreman but :+1:

File: README.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ To run the app, if you installed the optional dependencies you can run `nf start
...
 If you didn't install `webpack` globally, you can run it by `node_modules/.bin/webpack --watch`.
 
+#### Building the app
+To build the app for uploading into a Zendesk account, run:
+```
+npm run build
+zat package --path=./dist
+```
+
+Alternatively, you can use ZAT to create and update the app directly to your account via the API.
```
Comment (danielbreves) [inline README.md]: Can you add a description or link on how to do this here?

**Comment by adammw** [comment]
💇 @danielbreves ?

**Comment by danielbreves** [comment]
👍

**Pull request merged**